Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1456,13 +1497,18 @@ void GridTrackSizingAlgorithm::resolveIntrinsicTrackSizes()
     Vector<GridItemWithSpan> itemsSortedByIncreasingSpan;
     Vector<GridItemWithSpan> itemsCrossingFlexibleTracks;
     SingleThreadWeakHashSet<RenderBox> itemsSet;
+    SingleThreadWeakListHashSet<RenderBox> indefiniteItems;
```

Comment:
Same suggestion for the naming here since we'll be using it for a masonry specific purpose

Reply:
Updated to masonryIndefiniteItems.